src/common/extension-apis.ts

```diff
@@ -0,0 +1,14 @@
+/** Refers to {@link chrome} or `browser` in firefox */
+export const browser = globalThis.chrome || (globalThis as any).browser;
+
+if (!browser) {
+  throw new Error('chrome or browser not available in current context.');
+}
+
+export function getExtensionId() {
+  try {
+    return browser.runtime.id;
```

Comment:
This appears to work in Safari Web Extensions. Is there a browser this would throw in? https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/runtime/id has Chrome, Edge, FF, and Safari all supporting it in various forms.

Reply:
It's improbable that it will cause an error. I don't have a mac so I'm unsure about Safari <14 versions.

Reply:
fwiw InboxSDK's .browserslistrc notes that we only support back to Safari 15.4. I'd like to only support Safari 16+, but for now, our main product still supports Safari 15.4.